app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/notification/AppTpDisabledContentPlugin.kt

```diff
@@ -0,0 +1,69 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.mobile.android.vpn.service.notification
+
+import android.content.Context
+import androidx.core.app.NotificationCompat
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.mobile.android.vpn.AppTpVpnFeature
+import com.duckduckgo.mobile.android.vpn.R
+import com.duckduckgo.mobile.android.vpn.VpnFeaturesRegistry
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.NotificationContent
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.NotificationPriority
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.NotificationPriority.NORMAL
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.Type
+import com.duckduckgo.mobile.android.vpn.service.VpnReminderNotificationContentPlugin.Type.DISABLED
+import com.duckduckgo.mobile.android.vpn.ui.notification.NotificationActionReportIssue
+import com.squareup.anvil.annotations.ContributesMultibinding
+import javax.inject.Inject
+
+@ContributesMultibinding(AppScope::class)
+class AppTpDisabledContentPlugin @Inject constructor(
+    private val context: Context,
+    private val vpnFeaturesRegistry: VpnFeaturesRegistry,
+) : VpnReminderNotificationContentPlugin {
+    private val notificationPendingIntent by lazy { getAppTPNotificationPressIntent(context) }
+    private val notificationActionPendingIntent by lazy { getAppTPStartIntent(context) }
+
+    override fun getContent(): NotificationContent? {
+        return NotificationContent(
+            isSilent = true,
+            shouldAutoCancel = null,
+            customViewLayout = R.layout.notification_device_shield_disabled,
+            onNotificationPressIntent = notificationPendingIntent,
+            notificationAction = listOf(
+                NotificationCompat.Action(
+                    R.drawable.ic_vpn_notification_24,
+                    context.getString(R.string.atp_EnableCTA),
+                    notificationActionPendingIntent,
+                ),
+                NotificationActionReportIssue.reportIssueNotificationAction(context),
+            ),
+        )
+    }
+
+    override fun getPriority(): NotificationPriority {
+        return NORMAL
+    }
+
+    override fun isActive(): Boolean {
+        return !vpnFeaturesRegistry.isFeatureRegistered(AppTpVpnFeature.APPTP_VPN)
```

Comment:
Why not returning `true` instead, or even better, removing the `isActive()` method altogether until we have a second plugin (?)
This slightly changes the logic we had before, it's only necessary when we have more than one (DISABLED) plugin and it can be the source of empty notifications (as we've seen internally) or in this case, no notification